tests/chunked-encode-large.rs

```diff
@@ -72,13 +70,8 @@ async fn chunked_large() -> Result<(), http_types::Error> {
     let port = test_utils::find_port().await;
     let server = task::spawn(async move {
         let mut app = tide::new();
-        app.at("/").get(|mut _req: tide::Request<()>| async {
-            let mut res = Response::new(StatusCode::Ok);
-            let body = Cursor::new(TEXT.to_owned());
-            res.set_body(Body::from_reader(body, None));
-            res.set_content_type(mime::PLAIN);
-            Ok(res)
-        });
+        app.at("/")
+            .get(|_| async { Ok(Body::from_reader(Cursor::new(TEXT), None)) });
```

Comment:
Isn't it possible to rewrite this as:

```suggestion
        app.at("/").get(|_| async { Ok(TEXT) });
```

Reply:
Same as above, this test is explicitly a chunked test, so making that change would change the purpose of the test